XamarinCommunityToolkit/Converters/NotEqualsConverter.shared.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    /// <summary>
+    /// Checks whether the incoming value doesn't equal the provided parameter.
+    /// </summary>
+    public class NotEqualsConverter : IValueConverter
```

Comment:
**NotEqualConverter** ? Seems *s* is redundant here.

Reply:
I mainly named it that to indicate that it's the exact opposite of the `EqualsConverter`. I've also considered naming it `UnequalsConverter` but I suppose this could also be a greater discussion regarding the naming of inverse converters?

- `ListIsNullOrEmptyConverter` and:
   - `NotListIsNullOrEmptyConverter` vs.
   - `ListIsNotNullOrEmptyConverter`

Reply:
@sthewissen I mean Equal**s** is a verb, but Equal is an adjective.

EqualConverter and NotEqualConverter.